http-clients/src/test/java/org/triplea/http/client/web/socket/WebSocketConnectionTest.java

```diff
@@ -104,20 +142,27 @@ void tearDown() {
     @Test
     @DisplayName("Verify connect initiates connection and starts the pinger")
     void connectWillInitiateConnection() throws Exception {
-      givenWebSocketConnects(true);
-
-      final boolean connected =
-          webSocketConnection.connect(webSocketConnectionListener, errorHandler).get();
+      final HttpClient httpClient = mockHttpClient();
```

Comment:
Why `mockHttpClient()` instead of using `@Mock HttpClient httpClient`?

Reply:
AFAIK the mockito strict stups policy prevents declaring mocks that might not be used by all tests.
I'll create a nested test case to clean that up a bit